resources/sdk/purecloudgo/templates/api.mustache

```diff
@@ -81,29 +81,8 @@ func (a {{classname}}) {{nickname}}({{#allParams}}{{paramName}} {{{dataType}}}{{
 	for key := range a.Configuration.DefaultHeader {
 		headerParams[key] = a.Configuration.DefaultHeader[key]
 	}
-	{{#hasQueryParams}}
-	var collectionFormat string
-	{{#queryParams}}
-	{{#isListContainer}}
-	collectionFormat = "{{#collectionFormat}}{{collectionFormat}}{{/collectionFormat}}"
-	if collectionFormat == "multi" {
-		for _, value := range {{paramName}} {
-			queryParams["{{paramName}}"] = value
-		}
-	} else {
-		queryParams["{{paramName}}"] = a.Configuration.APIClient.ParameterToString({{paramName}}, collectionFormat)
-	}
-	{{/isListContainer}}
-	{{^isListContainer}}
-	collectionFormat = ""
-	if str, ok := interface{}({{paramName}}).(string); ok {
-		if str != "" {
```

Comment:
I think removing this empty string check for non-list params will break API calls that are relying on it to avoid adding optional query parameters. This should remain until the optional parameter changes to switch to pointers are in.

Reply:
Nevermind, it looks like this logic was moved to apiclient line 85.